src/DssSpell.sol

```diff
@@ -52,157 +66,106 @@ contract DssSpellAction is DssAction {
-        DssExecLib.sendPaymentFromSurplusBuffer(MHONKASALOTEEMULAU,   9_484);
-        // GFX Labs - 8903 DAI - 0xa6e8772af29b29B9202a073f8E36f447689BEef6
-        DssExecLib.sendPaymentFromSurplusBuffer(GFXLABS,              8_903);
-        // JustinCase - 7235 DAI - 0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473
-        DssExecLib.sendPaymentFromSurplusBuffer(JUSTINCASE,           7_235);
-        // London Business School Blockchain - 3798 DAI - 0xB83b3e9C8E3393889Afb272D354A7a3Bd1Fbcf5C
-        DssExecLib.sendPaymentFromSurplusBuffer(LBSBLOCKCHAIN,        3_798);
-        // CalBlockchain - 3421 DAI - 0x7AE109A63ff4DC852e063a673b40BED85D22E585
-        DssExecLib.sendPaymentFromSurplusBuffer(CALBLOCKCHAIN,        3_421);
-        // Blockchain@Columbia - 2851 DAI - 0xdC1F98682F4F8a5c6d54F345F448437b83f5E432
-        DssExecLib.sendPaymentFromSurplusBuffer(BLOCKCHAINCOLUMBIA,   2_851);
-        // Frontier Research LLC - 2285 DAI - 0xA2d55b89654079987CF3985aEff5A7Bd44DA15A8
-        DssExecLib.sendPaymentFromSurplusBuffer(FRONTIERRESEARCH,     2_285);
-        // Chris Blec - 1334 DAI - 0xa3f0AbB4Ba74512b5a736C5759446e9B50FDA170
-        DssExecLib.sendPaymentFromSurplusBuffer(CHRISBLEC,            1_334);
-        // CodeKnight - 355 DAI - 0x46dFcBc2aFD5DD8789Ef0737fEdb03489D33c428
-        DssExecLib.sendPaymentFromSurplusBuffer(CODEKNIGHT,             355);
-        // ONESTONE - 342 DAI - 0x4eFb12d515801eCfa3Be456B5F348D3CD68f9E8a
-        DssExecLib.sendPaymentFromSurplusBuffer(ONESTONE,               342);
-        // pvl - 56 DAI - 0x6ebB1A9031177208A4CA50164206BF2Fa5ff7416
-        DssExecLib.sendPaymentFromSurplusBuffer(PVL,                     56);
-        // ConsenSys - 33 DAI - 0xE78658A8acfE982Fde841abb008e57e6545e38b3
-        DssExecLib.sendPaymentFromSurplusBuffer(CONSENSYS,               33);
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmYUi9Tk
+
+        // Increase WSTETH-B Stability Fee to 0.25%
+        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_PT_TWO_FIVE_PCT_RATE, true);
+
+        // Increase Compound v2 D3M Maximum Debt Ceiling to 20 million
+        // Set Compound v2 D3M Target Available Debt to 5 million DAI (this might already be the case)
+        DssExecLib.setIlkAutoLineParameters("DIRECT-COMPV2-DAI", 20 * MILLION, 5 * MILLION, 12 hours);
+
+        // Increase the USDP PSM tin to 0.2%
+        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", 20 * WAD / 10000);   // 20 BPS
+
+
+        // MKR Transfer for CES
+        // https://vote.makerdao.com/polling/QmbNVQ1E
+
+        // CES-001 - 96.15 MKR - 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da
+        GemLike(DssExecLib.mkr()).transfer(CES_WALLET, 96.15 ether); // ether as solidity alias
+
+
+        // Cage DIRECT-AAVEV2-DAI
+        // https://forum.makerdao.com/t/housekeeping-tasks-for-next-executive/19472
+
+        CageLike(MCD_JOIN_DIRECT_AAVEV2_DAI).cage();
```

Comment:
Why are we caging it?

Reply:
Exec says so:

In preparation for a new Aave D3M deployment if this executive proposal passes it will `cage` the old Aave D3M (DIRECT-AAVEV2-DAI).

https://raw.githubusercontent.com/makerdao/community/2d14219f8850d2261c1e65b7faa7cfecd138951f/governance/votes/Executive%20vote%20-%20January%2027%2C%202023.md

Reply:
> Why are we caging it?

Do you see an issue with this? We're shutting the module down and cutting it off from the system, why wouldn't we make `live = 0`?

Reply:
> Exec says so:  
>In preparation for a new Aave D3M deployment if this executive proposal passes it will cage the old Aave D3M (DIRECT-AAVEV2-DAI).

@The-Arbiter For complete clarity it was a PE call to cage the old D3M. See Derek's forum post: https://forum.makerdao.com/t/housekeeping-tasks-for-next-executive/19472